src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        super.setColor(color);
+    }

```

Comment:
```suggestion
    public Circle(int radius, String color) {
        this.radius = radius;
        setColor(color);
    }

```

why do we need `super` keyword here?